sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -167,68 +167,123 @@ ConfigurationAsyncClient client = new ConfigurationClientBuilder()
         .connectionString(appConfigConnectionString)
         .buildAsyncClient();
 
-client.listSettings(new SettingSelector().setLabels(periodicUpdateLabel))
+client.listConfigurationSettings(new SettingSelector().setLabels(periodicUpdateLabel))
     .subscribe(setting -> updateConfiguration(setting));
 ```
 
 ## Examples
 
-The following sections provide several code snippets covering some of the most common Configuration Service tasks, including:
-- [Create a Configuration Setting](#create-a-Configuration-Setting)
-- [Retrieve a Configuration Setting](#retrieve-a-Configuration-Setting)
-- [Update an existing Configuration Setting](#update-an-existing-Configuration-Setting)
-- [Delete a Configuration Setting](#delete-a-Configuration-Setting)
-
-### Create a Configuration Setting
-
-Create a Configuration Setting to be stored in the Configuration Store. There are two ways to store a Configuration Setting:
-- addSetting creates a setting only if the setting does not already exist in the store.
-- setSetting creates a setting if it doesn't exist or overrides an existing setting.
+The following sections provide several code snippets covering some of the most common configuration service tasks, including:
+- [Create a configuration setting][sample_hello_world]
+- [Retrieve a configuration setting][sample_hello_world]
+- [Update an existing configuration setting][sample_hello_world]
+- [Delete a configuration setting][sample_hello_world]
+- [List configuration settings with multiple keys][sample_list_configuration_settings]
+- [List revisions of multiple configuration settings][sample_read_revision_history]
+- [Set a configuration setting to read only][sample_read_only]
+- [Clear read only from a configuration setting][sample_read_only]
+- [Conditional request a configuration setting][sample_conditional_request]
+
+### Create a Configuration Client
```

Comment:
Your sentence style casing should also apply to these headers.
Create a configuration client
Create a configuration setting, etc.